vars/build.groovy

```diff
@@ -35,7 +35,8 @@ def call(Map args) {
       } catch (e) {
         status = "fail"
       } finally {
-        Events.emit(["build.end", "build.${status}"], [status: status, namespace: namespace])
+        Events.emit(["build.end", "build.${status}"],
+                    [status: status, namespace: namespace, gitURL: gitURL, commitHash: commitHash])
```

Comment:
how about `[ ... git: [url: gitURL, commit: commit], ..]` ?
So that the caller can use `args.git.url ` and `args.git.commit` and can pass `args.git` to methods requiring it

Reply:
:heart: